src/NuGet.Frameworks/FrameworkNameProvider.cs

```diff
@@ -702,5 +714,28 @@ public bool TryGetSubSetFrameworks(string frameworkIdentifier, out IEnumerable<s
             subSetFrameworks = null;
             return false;
         }
+
+        // Order the frameworks if rules exist, this is used to break ties between equally compatible frameworks
+        // EX: UAP -> Win81, WPA81
+        public int CompareFrameworks(NuGetFramework x, NuGetFramework y)
+        {
+            // Using the ordered list of frameworks choose the framework that matches first
+            if (!StringComparer.OrdinalIgnoreCase.Equals(x.Framework, y.Framework))
```

Comment:
You could do `string.Equals(x.Framework, y.Framework, StringComparison.OrdinalIgnoreCase)` instead